.github/workflows/notebook-pr.yaml

```diff
@@ -47,7 +47,7 @@ jobs:
           echo ci/ >> .gitignore
 
       - name: Install dependencies
-        if: "!contains(env.COMMIT_MESSAGE, 'skip ci')"
+        if: "contains(env.COMMIT_MESSAGE, 'ci:execute')"
```

Comment:
I think you probably want both conditions here, right?

Reply:
i.e., install the dependencies if you are (a) not skipping ci AND (b) executing the notebooks

Reply:
correct I m fixing it